docs/DeveloperGuide.adoc

```diff
@@ -34,24 +34,23 @@ The *_Architecture Diagram_* given above explains the high-level design of the A
 
 [TIP]
 The `.puml` files used to create diagrams in this document can be found in the link:{repoURL}/docs/diagrams/[diagrams] folder.
-Refer to the <<UsingPlantUml#, Using PlantUML guide>> to learn how to create and edit diagrams.
 
 `Main` has two classes called link:{repoURL}/src/main/java/seedu/address/Main.java[`Main`] and link:{repoURL}/src/main/java/seedu/address/MainApp.java[`MainApp`]. It is responsible for,
 
-* At app launch: Initializes the components in the correct sequence, and connects them up with each other.
-* At shut down: Shuts down the components and invokes cleanup method where necessary.
+* Initializes the components in the correct sequence, and connects them up with each other.
+* Shuts down the components and invokes cleanup method where necessary.
 
 <<Design-Commons,*`Commons`*>> represents a collection of classes used by multiple other components.
-The following class plays an important role at the architecture level:
+`LogsCenter` plays an important role at the architecture level. Here is how it works:
 
-* `LogsCenter` : Used by many classes to write log messages to the App's log file.
+* Uses `LogsCenter` to write log messages to the App's log file.
```

Comment:
```suggestion
* `LogsCenter` is used by many classes to write log messages to the App's log file.
```

because now i'm not sure who uses it if it starts with "Uses"

Reply:
Okay. But I think this description still need to change because other descriptions describe how the high-level design works, but this one seems like how the inner part works.